### ISSUE TITLE
Some abilities missing + PvP talents

### ISSUE THREAD

**Issue by Llamatron2112** [Silencer2K/wow-action-bar-profiles#2]
First of all thanks for this great addon, been looking for something like that.

But I ran into a couple of problems:
- It doesn't see some abilities, and removes them from the bar, at least that's what it did with my hunter's traps, maybe because they are in a "submenu" in the spellbook.

- It doesn't save nor see honor talents, and removes them from the action bars.

**Comment by Sharagran** [comment]
Have the same issue, it cant place Cloudburst Totem in my actionbar and says "Spell not found: [Cloudburst Totem]". Its probably because Cloudburst Totem replaces Healing Stream Totem. If I edit my profile in ActionBarProfiles.lua and change the actionbar spell from Cloudburst Totem to Healing Stream Totem then it works. Unfortunately I have no experience with WoW addons and have no idea how this issue could be fixed within the code.

**Comment by FuryusSvetlo** [comment]
Let's call your resto shammy that you're trying to "use" the profile on you "target" shammy, and the shammy you were logged into when you "saved" that profile your "source" shammy.

If your target resto shammy does not have talent 6/3 (Cloudburst Totem) selected at the time at which you Use that profile, then you'll get the error you indicated above.

This is generally true for all uses of this addon: the addon uses the built-in WoW function for placing spells on your action bars... and it can only place spells on your action bars that you currently "know".

Hope that helps, and happy to hear feedback to the contrary if observed.

**Comment by Sharagran** [comment]
Yeah I guess that would work too. I already worked around it by just creating a macro that casts the spell instead of using the actual spell.
I just found it a bit confusing that it doesnt seem to work on the first attempt, even though the addon changes the talents first before it places the skills.
Anyways thanks for your help and time.
